packages/runtime/container-runtime/src/deltaManagerProxies.ts

```diff
@@ -194,3 +195,19 @@ export class DeltaManagerSummarizerProxy extends BaseDeltaManagerProxy {
 		this.isSummarizerClient = this.deltaManager.clientDetails.type === summarizerClientType;
 	}
 }
+
+export class DeltaManagerPendingOpsProxy extends BaseDeltaManagerProxy {
+	public get minimumSequenceNumber(): number {
```

Comment:
This is only one method where MSN shows up. I think we need to review API and see what else needs to be faked.
For example, DM.lastMessage.minimumSequenceNumber should match what this thing returns (although it's worth pointing out that it will "magically" change its value once we reconnect, as MSN will move forward).

Reply:
I've updated this, do you think there are other parts? I did an investigation based on looking at `minimumSequenceNumber` and `ISequenceDocumentMessage` usage in the codebase.

Reply:
My investigation was based on two main paths - summarization and processing ops. Do you think the `ScheduleManager` needs to have this "proxied" view of the world?